src/micromamba/clean.cpp

```diff
@@ -45,17 +53,31 @@ set_clean_command(CLI::App* subcom)
     init_clean_parser(subcom);
 
     subcom->callback([&]() {
+        load_configuration();
+
         auto& ctx = Context::instance();
+        auto& config = Configuration::instance();
+
+        auto& clean_all = config.at("clean_all").value<bool>();
+        auto& clean_index = config.at("clean_index_cache").value<bool>();
+        auto& clean_pkgs = config.at("clean_packages").value<bool>();
+        auto& clean_tarballs = config.at("clean_tarballs").value<bool>();
 
-        load_clean_options(ctx);
+        if (!(clean_all || clean_index || clean_pkgs || clean_tarballs))
+        {
+            std::cout << "Nothing to do." << std::endl;
+            return;
+        }
 
-        Console::print("Cleaning up ... ");
+        LOG_INFO << "Collect information...";
```

Comment:
I think it's fine to print some output by default with this command so that the user knows something is going on.